doc/.vuepress/config.js

```diff
@@ -77,10 +77,19 @@ module.exports = {
                       'actor',
                       'crypto',
                       'protobuf',
-                      'xml',
                       'parser',
                       'values',
 		      'mime/',
+		      {
+			  title: "Markup Languages",
+			  path: "/reference/std/markup/",
+			  children: [
+			      "markup/sxml/",
+			      "markup/sxml/xml",
+			      "markup/sxml/html/",
+			      "markup/sxml/tal/"
+			  ]
```

Comment:
do we need the sxml directory? why not just put the directly under markup?

Reply:
Like mentioned, there will be other non-sxml things there before 0.19 so I figure that's a decent classification.